p2p_wallet/Scenes/Main/Swap/Providers/JupiterTokensRepository.swift

```diff
@@ -3,42 +3,58 @@ import Combine
 import SolanaSwift
 import Resolver
 
+/// Tokens data returned by Jupiter
 struct JupiterTokensData {
+    /// List of Jupiter's supported tokens
     let tokens: [SwapToken]
+    /// Wallets owned by user
     let userWallets: [Wallet]
 }
 
+/// Repository that handle JupiterTokens
 protocol JupiterTokensRepository {
-    var status: AnyPublisher<JupiterDataStatus, Never> { get }
-
```

Comment:
I think `status` doesn't clearly indicate its type `AnyPublisher`. When thinking about `status`, we may find it is closer to `JupiterDataStatus` than `AnyPublisher<JupiterDataStatus, Never>`.
So my suggestion is changing it to `statusPublisher` like some properties in standard Apple iOS SDK